chpl/chpl-service/src/main/java/gov/healthit/chpl/certificationId/Validator2015.java

```diff
@@ -100,6 +103,9 @@ protected boolean isCPOEValid() {
                 return true;
             }
         }
+        SortedSet<String> missing1 = new TreeSet<String>();
```

Comment:
Shouldn't use variable name with this format. There's no other variable in this method named `missing`, so just drop the "1"